### ISSUE TITLE
Arithmetic operations `VariableRef--ParametrizedAffExpr{C}` delete variables

### ISSUE THREAD

**Issue by jd-lara** [JuliaStochOpt/ParameterJuMP.jl#42]
When adding a variable to a `ParameterJuMP.ParametrizedAffExpr{Float64}` the function 

https://github.com/JuliaStochOpt/ParameterJuMP.jl/blob/23b0250d8091dfd2d9a651e558399de6057f002d/src/operators.jl#L119

Eliminates previous variables in the internal `JuMP.GenericAffExpr{C,JuMP.VariableRef}`

```Julia
model = ModelWithParams()
x = @variable(model)
y = @variable(model)
a = Parameter(model, 10)
b = Parameter(model, 11)

exp1 = x + y + a
exp2 = exp1 + b

julia> typeof(exp1)
ParameterJuMP.ParametrizedAffExpr{Float64}

julia> exp1.v.terms
OrderedCollections.OrderedDict{VariableRef,Float64} with 2 entries:
  noname => 1.0
  noname => 1.0

julia> exp1 = exp1 + y;

julia> exp1.v.terms
OrderedCollections.OrderedDict{VariableRef,Float64} with 1 entry:
  noname => 1.0

exp3 = x + y + a
z = @variable(model)
exp3 = exp3 + z

julia> exp3.v.terms
OrderedCollections.OrderedDict{VariableRef,Float64} with 1 entry:
  noname => 1.0

```

**Comment by joaquimg** [comment]
these method are the problem:
```julia
Base.:(+)(lhs::JuMP.VariableRef, rhs::PAE{C}) where {C} = PAE{C}(GAEv{C}(zero(C), lhs => 1.),copy(rhs.p))
Base.:(-)(lhs::JuMP.VariableRef, rhs::PAE{C}) where {C} = PAE{C}(GAEv{C}(zero(C), lhs => 1.),-rhs.p)
```
we need to start adding tests for parameteric affine expressions

**Comment by joaquimg** [comment]
I will fix

**Comment by jd-lara** [comment]
agreed on the testing, can help with that once I am done with integrating ParameterJuMP in PowerSimulations.

**Comment by joaquimg** [comment]
solved by #44
